DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/schedulev2/ScheduleV2Migration.java

```diff
@@ -196,16 +196,48 @@ public static TestSchedule createHMSchedule(String uId, String scheduleJson) thr
     }
 
     public static void
-    createV2Schedule(SageV1Schedule v1Schedule, String uId, String sId) throws IOException {
+    createV2Schedule(String arcId, SageV1Schedule v1Schedule, String uId, String sId) throws IOException {
 
         String iANATimezone = getTimezone(v1Schedule);
+        System.out.println("User timezone is " + iANATimezone);
+
+        DateTime studyStart = SageScheduleController.Companion
+                .createDateTime(v1Schedule.getStudyBursts().get(0).getStartDate(), iANATimezone)
+                // Putting this date at noon, gives Bridge the most flexibility with
+                // how it adjusts local times and time zones to get the correct start day
+                .withTimeAtStartOfDay().plusHours(12);
+
         for (int i = 0; i < v1Schedule.getStudyBursts().size(); i++) {
             SageV1StudyBurst studyBurst = v1Schedule.getStudyBursts().get(i);
 
             // The new V2 Sage scheduling does not include the practice test, and so
             // the study as a whole, and the first study burst schedule starts.
             DateTime startDate = SageScheduleController.Companion
-                    .createDateTime(studyBurst.getStartDate());
+                    .createDateTime(studyBurst.getStartDate(), iANATimezone)
+                    // Putting this date at 1 PM, gives Bridge the most flexibility with
+                    // how it adjusts local times and time zones to get the correct start day
+                    // Add one additional hour to account for any daylight savings issues
+                    .withTimeAtStartOfDay().plusHours(13);
```

Comment:
Here you chose 1pm and above noon?

Reply:
I noticed that daylight savings shifted the days calculation by an hour, so I wanted to make sure that the days since study start didn't lose a day as it doesn't return fractional days.